From 4a5addd98c20025f655ecb768b30e80260cea763 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 14 Jan 2020 19:16:43 +0100
Subject: GCC: media_log_properties_helper fix

GCC does not allow extra qualification on the declaration of specializers.

Bug: 819294
Change-Id: I1915c5a8f5a778fc5515cff11f51a4bcca80236b
---
 media/base/media_log_properties_helper.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/base/media_log_properties_helper.h b/media/base/media_log_properties_helper.h
index 95ff70a82be0..549b0037da71 100644
--- a/media/base/media_log_properties_helper.h
+++ b/media/base/media_log_properties_helper.h
@@ -83,7 +83,7 @@ struct MediaLogPropertyTypeConverter<std::vector<T>> {
 
 // Specializer for sending AudioDecoderConfigs to the media tab in devtools.
 template <>
-struct internal::MediaLogPropertyTypeConverter<media::AudioDecoderConfig> {
+struct MediaLogPropertyTypeConverter<media::AudioDecoderConfig> {
   static base::Value Convert(const AudioDecoderConfig& value) {
     base::Value result(base::Value::Type::DICTIONARY);
     result.SetStringKey("codec", GetCodecName(value.codec()));
@@ -114,7 +114,7 @@ struct internal::MediaLogPropertyTypeConverter<media::AudioDecoderConfig> {
 
 // Specializer for sending VideoDecoderConfigs to the media tab in devtools.
 template <>
-struct internal::MediaLogPropertyTypeConverter<VideoDecoderConfig> {
+struct MediaLogPropertyTypeConverter<VideoDecoderConfig> {
   static base::Value Convert(const VideoDecoderConfig& value) {
     base::Value result(base::Value::Type::DICTIONARY);
     result.SetStringKey("codec", GetCodecName(value.codec()));
-- 
2.20.1

